python/py-asyncpg/Portfile

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        MagicStack asyncpg 0.22.0 v
+revision            0
+name                py-${github.project}
+categories-append   devel
+fetch.type          git
+
+license             Apache2
+maintainers         nomaintainer
+platforms           darwin
+
+description         An asyncio PostgreSQL driver
+
+long_description    asyncpg is a database interface library designed \
+                    specifically for PostgreSQL and \
+                    Python/asyncio. asyncpg is an efficient, clean \
+                    implementation of PostgreSQL server binary \
+                    protocol for use with Python's asyncio \
+                    framework.
+
+python.versions     37 38 39
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-pip \
+                    port:py${python.version}-setuptools \
+                    port:py${python.version}-wheel
+
+    depends_lib-append \
+                    port:py${python.version}-cython \
```

Comment:
I think you're missing a dependency on `py-typing_extensions`....

https://github.com/MagicStack/asyncpg/blob/161607603adae789d4392502859cc7bbdec7240b/setup.py#L295

Reply:
Thanks, fixed.